frontend/src/modules/EditZing/Components/EmailModalContent.tsx

```diff
@@ -0,0 +1,153 @@
+import { useState } from 'react'
+import { EmailModalContentProps } from '../Types/ComponentProps'
+import {
+  ModalContainer as ContentContainer,
+  RecipientsContainer as InfoContainer,
+} from '../Styles/Email.style'
+import { EmailTemplateButtons } from './EmailTemplateButtons'
+import { EmailPreview } from './EmailPreview'
+import { Box, Button, Typography } from '@mui/material'
+import { TemplateName } from 'EditZing/utils/emailTemplates'
+import SendIcon from '@mui/icons-material/Send'
+
+export const EmailModalContent = ({
+  selectedGroups,
+  courseNames,
+  isEmailing,
+  setIsEmailing,
+}: EmailModalContentProps) => {
+  const [selectedTemplate, setSelectedTemplate] = useState<TemplateName>(
+    '' as TemplateName
+  )
```

Comment:
I feel like the `'' as TemplateName` is abusive of TypeScript because we are basically convincing TypeScript that whatever's on the left hand side will be of the correct type, except here we already know `''` is not part of the enum type.